lighthouse-core/gather/connections/connection.js

```diff
@@ -126,15 +117,14 @@ class Connection {
   }
 
   /**
-   * @param {string} method
-   * @param {object=} params
-   * @protected
+   * @param {LH.Protocol.RawEventMessage} eventMessage
    */
-  emitNotification(method, params) {
+  emitProtocolEvent(eventMessage) {
     if (!this._eventEmitter) {
       throw new Error('Attempted to emit event after connection disposed.');
     }
-    this._eventEmitter.emit('notification', {method, params});
+
+    this._eventEmitter.emit('protocolevent', eventMessage);
```

Comment:
discussed this with @paulirish earlier. Since this event emitter can *only* emit protocol events (and `Driver` is the only receiver), it makes sense to make the single event it handles be something more descriptive than "notification" :)